src/app/tests/TestCommandInteraction.cpp

```diff
@@ -326,7 +343,7 @@ class TestCommandInteraction
     static void TestCommandHandlerEncodeSimpleStatusCode(nlTestSuite * apSuite, void * apContext);
     static void TestCommandHandlerWithSendEmptyResponse(nlTestSuite * apSuite, void * apContext);
 
-    static void TestCommandHandlerWithProcessReceivedEmptyDataMsg(nlTestSuite * apSuite, void * apContext);
+    static void TestCommandHandlerWithOnInvokeReceivedEmptyDataMsg(nlTestSuite * apSuite, void * apContext);
```

Comment:
TODO a few tests where `CommandHandler` does not have something that implements `CommandResponderInterface`

* TestCommandHandlerWithoutResponderCallingAddStatus
* TestCommandHandlerWithoutResponderCallingAddStatus
* TestCommandHandlerWithoutResponderCallingRawApis